Formula/nfdump.rb

```diff
@@ -11,8 +11,16 @@ class Nfdump < Formula
     sha256 "5bbd8ec9a51cde3c6ed0aaeb1b77a4f2ba59bea99d5304fa1415a7e2e0d5280b" => :mountain_lion
   end
 
+  option "with-readpcap", "Read netflow packets from a give pcap_file instead of the network. Is intended for debugging only."
```

Comment:
I see your point @apjanke, but being that it is an optional debugging feature, it would seem more logical to me to include it as an option. As a user of nfcapd/nfdump (and the original author of this formula), I certainly wouldn't have expected that option included by default.